.scripts/transifex/parse.js

```diff
@@ -0,0 +1,39 @@
+// Open https://www.transifex.com/languages/
+// Open console
+// Run this:
+
+i18n_codes = (function(){
+	var codes = {};
+	$("#txlanguage-list li").each(function () {
```

Comment:
This kind of code makes me nervous because it seems brittle. This isn't run on every release is it?